elastic-search-support/src/test/resources/integration/case_type_mapping.json

```diff
@@ -32,10 +32,12 @@
       }
     },
     "created_date": {
-      "type": "date"
+      "type": "date",
+      "ignore_malformed": true
```

Comment:
Good this file will now match what is actually expected based on the `application.yml` rather than outdated overrides for tests, should help potential regressions and force this to be updated for new field types 👍